getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -1,3 +1,5 @@
+:article_outdated: True
```

Comment:
Why this, it was marked not out of date since the 4.2 version, what about the 4.3 version is outdated?

Reply:
> Why this, it was marked not out of date since the 4.2 version, what about the 4.3 version is outdated?

I'm not sure where that came from. I'll try a new commit from current source file in a bit.

Reply:
It was already on there. I have removed the line.